### ISSUE TITLE
Show local like player for YouTube audio only mode

### ISSUE THREAD

**Issue by catgangdev** [namidaco/namida#19]
Hello , when we select YouTube audio only mode it shows player view kinda like video player. Can you make player view like local player. Also if we select audio only mode it will show music only not other videos. I want to use it as a music player alternative to YouTube music . Thanks in advance

**Comment by MSOB7YY** [comment]
TLDR: currently you can download the audio and it will be added automatically to ur library, (if its not ur first time time listening, then it will be instantly added from cache)
----
it might sound like a good idea but currently local queue and youtube queue are managed differently, u either play from youtube or from local, i had plans to support mixed queue/playlist but that would need lot of boilerplate and sessions saving will not work properly without internal logic modifications
refer to [local miniplayer](https://github.com/namidaco/namida/blob/main/lib/packages/miniplayer.dart) and [youtube miniplayer](https://github.com/namidaco/namida/blob/main/lib/youtube/youtube_miniplayer.dart) to understand more, both are completely different types with different apis and ways to display stuff

leaving open since it might be possible later

**Comment by catgangdev** [comment]
I got your point but I rarely listen offline music. I was looking for a online music player alternative.
Maybe in future I will use it. Thanks for your assistance

**Comment by Red6785** [comment]
If you want an online music player, possibly check out BlackHole.

**Comment by MSOB7YY** [comment]
@catgangdev good news, been working on this, its gonna be out soon!

**Comment by MSOB7YY** [comment]
just a reminder this is available in v2.0.6
